tests/test_rio_merge.py

```diff
@@ -598,6 +599,10 @@ def test_merge_filenames(tiffs):
     inputs.sort()
     merge(inputs, res=2)
 
+    inputs = [Path(x) for x in tiffs.listdir()]
+    inputs.sort()
+    merge(inputs, res=2)
+
```

Comment:
@fwfichtner I must insist that this be a separate test case. Copying `test_merge_filenames()` and renaming it will be fine.

Reply:
Right, thanks for the feedback!